lib/manager/components/version/FeedVersionDetails.js

```diff
@@ -85,7 +105,17 @@ export default class FeedVersionDetails extends Component<Props> {
                 </DropdownButton>
               }
               <Button
-                disabled={isPublished || processing || !userCanManageFeed}
+                disabled={
+                  !gtfsPlusValidation ||
+                  gtfsPlusValidation.issues.length > 0 ||
+                  !gtfsPlusValidation.published ||
+                  !version.validationResult ||
+                  !version.validationResult.error_counts ||
+                  hasBlockingIssue ||
+                  isPublished ||
+                  processing ||
+                  !userCanManageFeed
+                }
```

Comment:
It'd be nice for there to be a tooltip or some explainer on why this button is disabled since there are a lot of reasons why.

Reply:
I see there is something below if there are blocking issues, but there are more reasons up here that cause the button to be disabled.

Reply:
Many of the reasons listed here (which are not shown in the message below) are more so related to the fact that things have not fully loaded yet.